src/routes/messages.js

```diff
@@ -16,10 +16,11 @@ module.exports = (() => {
     *
 	* @apiHeader {String} token Authorization Bearer Token.
     * @apiParam {String} mid Message's id.
+	* @apiParam (Request body) {String} emoji Emoji reaction.
     *
-    * @apiSuccess (Created 201) {Number} id Message ID.
-    * @apiSuccess (Created 201) {Number} id User ID.
```

Comment:
These were referred to with the same name, so only Message id showed up in the docs.